### PR TITLE
auro-flight: new departure/arrival reroute support for IRROPS

### PR DESCRIPTION
**Pull request by gusnaughton**
# Alaska Airlines Pull Request

Adds a new arrival and departure station attribute for displaying rerouted flight plans.

**Fixes:** #12 

## Type of change:

Please delete options that are not relevant.

- [x] New capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by KyleJones2** [comment]
UI Approved for footer slot reroutes.

MicrosoftTeams-image (52)

File: docs/api.md [hunk 1]
```diff
@@ -18,18 +18,22 @@ DoT: STATION SIZE AND COLOR MUST BE IDENTICAL TO DISCLOSURE SIZE AND COLOR!
...
-| `arrivalTime`      | `arrivalTime`      | `String` | Time of arrival, ex: `9:06 pm`   |
-| `departureStation` | `departureStation` | `String` | Station of departure, ex: `PVD`  |
-| `departureTime`    | `departureTime`    | `String` | Time of departure, ex: `5:36 am` |
+| Property              | Attribute             | Type     | Description                              |
+|-----------------------|-----------------------|----------|------------------------------------------|
+| `arrivalStation`      | `arrivalStation`      | `String` | Station of arrival, ex: `SEA`            |
+| `arrivalTime`         | `arrivalTime`         | `String` | Time of arrival, ex: `9:06 pm`           |
+| `departureStation`    | `departureStation`    | `String` | Station of departure, ex: `PVD`          |
+| `departureTime`       | `departureTime`       | `String` | Time of departure, ex: `5:36 am`         |
+| `newArrivalStation`   | `newArrivalStation`   | `String` | Station of rerouted arrival, ex: `AVP`   |
```
Comment (blackfalcon) [inline docs/api.md:27]: Just catching this now, but we are not using `ex` for example. Please update to `e.g.`

File: src/style-flight-main.scss [hunk 2]
```diff
@@ -72,3 +72,9 @@
   display: inline-flex;
   flex-direction: column;
 }
+
+.rerouted-line-through{
```
Comment (blackfalcon) [inline src/style-flight-main.scss:76]:
```suggestion
.util_lineThrough {
```

File: src/style-flight-main.scss [hunk 3]
```diff
@@ -72,3 +72,9 @@
...
+  color: var(--auro-color-text-error-on-light);
+  text-decoration: line-through;
+  padding-right: 16px;
```
Comment (blackfalcon) [inline src/style-flight-main.scss:79]:
```suggestion
  padding-right: var(--auro-size-md);
```

File: src/style-flight-main.scss [hunk 4]
```diff
@@ -72,3 +72,9 @@
   display: inline-flex;
   flex-direction: column;
 }
+
+.rerouted-line-through{
+  color: var(--auro-color-text-error-on-light);
```
Comment (blackfalcon) [inline src/style-flight-main.scss:77]:
```suggestion
  color: var(--auro-color-alert-error-on-light);
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]
This demo confused me
Screen Shot 2021-05-23 at 11 37 44 AM

`newDepartureStation="PDX"` and yet PDX has the line-through? Is that correct?

**Comment by gusnaughton** [comment]
@blackfalcon  addressed feedback ty

File: src/auro-flight-main.js [hunk 5]
```diff
@@ -32,6 +34,8 @@ class AuroFlightMain extends LitElement {
       arrivalStation:   { type: String },
       departureTime:    { type: String },
       departureStation: { type: String },
+      oldDepartureStation: {type: String},
```
Comment (blackfalcon) [inline src/auro-flight-main.js:37]:
```suggestion
      reroutedDepartureStation: {type: String},
```

Playing with this, using `old` really felt lazy to me. What do you feel about updating to `rerouted`?

Looking at the screenshot attached to the issue, what about rerouted times? Don't those need to change too?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: src/auro-flight-main.js (hunk 5 above)
Comment (gusnaughton) [reply src/auro-flight-main.js:37]: Rerouted times are addressed in the departureHeader and arrivalHeader slots. I will rename the attribute rerouted.

**Comment by gusnaughton** [comment]
Update made, ready to roll.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**